scripts/release-notes/gen_ec_release_notes.py

```diff
@@ -160,6 +176,25 @@ def main():
         publish_release(new_release, not args.dry_run)
 
 
+def get_access_token_for_app(app_id, key_path, org, repo):
+    """Return an access token for a GitHub App."""
+    app_id = os.environ.get('APP_ID')
+    if not app_id:
+        raise RuntimeError('APP_ID is not set')
+    key_path = os.environ.get('CLIENT_KEY')
+    if not key_path:
+        raise RuntimeError('CLIENT_KEY is not set')
+    integration = github.GithubIntegration(
+        app_id,
+        pathlib.Path(key_path).read_text(encoding='utf-8'),
+    )
+    app_installation = integration.get_installation(GITHUB_ORG, GITHUB_REPO)
+    if app_installation is None:
+        sys.exit(f"Failed to get app_installation for {GITHUB_ORG}/{GITHUB_REPO}. " +
```

Comment:
Just curious: here it's `sys.exit` but in lines above it's `raise` - any difference? I think uncaught exception produces stacktrace?

Reply:
I left the sys.exit() in the original copy I made from your script. I've switched to RuntimeError for consistency. I prefer the exception because it lets me see where in the code the error is coming from.